src/components/Board.js

```diff
@@ -7,23 +7,88 @@ import Card from './Card';
 import NewCardForm from './NewCardForm';
 import CARD_DATA from '../data/card-data.json';
 
+const URL = "https://inspiration-board.herokuapp.com/boards/KTJ/cards"
+const deleteURL = "https://inspiration-board.herokuapp.com/cards/"
+
 class Board extends Component {
   constructor() {
     super();
 
     this.state = {
       cards: [],
+      errorMessage: '',
     };
   }
 
-  render() {
-    return (
-      <div>
-        Board
-      </div>
-    )
+  componentDidMount(){
+    axios.get(URL)
+    .then((response) => {
+      const card = response.data.map((info) => {
+
+        return info.card
+      })
+        this.setState({
+          cards: card
+        });
+      })
+      .catch((error) =>{
+        console.log(error.message);
+
+        this.setState({
+          errorMessage: error.message,
+      })
+    })
   }
 
+  deleteCard = (id) => {
+    axios.delete(`${deleteURL}${id}`)
+      .then((response) => {
+        let cardList  = this.state.cards.filter(element => element.id !== response.data.card.id)
+        this.setState({cards: cardList});
+      })
+      .catch((error) => {
+        console.log(error.message);
+      })
+  }
+
+  addCard = (card) =>{
+    axios.post(URL, card)
+    .then((response)=>{
+      card.id = response.data.card.id
+
+      const { cards} = this.state
+      cards.push(card)
+      this.setState({
+        cards
+      });
+    })
+    .catch((error) => {
+      console.log(error);
+      this.setState({
+          errorMessage: `Failure ${error.message}`,
+        })
+    })
+  }
+
+  render() {
+    const cards = this.state.cards.map((card, i) =>{
+    return <Card
+          key={i}
```

Comment:
I strongly suggest not using the index of the card as the  key, unless you have no other unique identifiers.  The `id` field would work much better here.